.github/CODEOWNERS

```diff
@@ -24,48 +24,36 @@
 # subdirectories.
 #/build/logs/      @doctocat
 
-/.circleci/        @bitaholic
-/.github/          @kjhman21
 /accounts/         @kjhman21
```

Comment:
@kjhman21 Could add @KimKyungup to accounts?
I would like to look bind/simulated backend.

Reply:
Could you add @jeonghonah to accounts?